### DISCUSSION TITLE
Disk Management

### DISCUSSION THREAD

**Discussion by mgrennan** [thehighnibble/thehighnibble.github.io#40] [Cromemco Z-1 replica]
How do you create blank floppies (dsk) and hard disks (hdd) files.  Then format them in each of the OSs.

**Comment by tomxp411** [comment]
Usually, I copy an existing image file of the type I want, then either erase all the files or run the appropriate format command for that OS. 

Since every OS is different, you'll have to refer to each OS's manual for the appropriate format command. On CP/M, I believe you can mount the disk in something other than the A drive and run FORMAT from the A drive, like this: 

FORMAT B:

**Reply by GrantMeStrength** [reply]
Check the "Plotting Spock" thread, as it mentions cpmtools which is the software that you can use to create disk images.

**Comment by udo-munk** [comment]
Also z80pack includes tools that let you create empty disk images in all supported formats.
